src/resources/views/crud/fields/repeatable.blade.php

```diff
@@ -16,10 +16,11 @@
       @include('crud::fields.inc.attributes')
   >
 
-  <div class="container-repeatable-elements">
```

Comment:
It's possible that this field is used more than once inside a form, and that other sortable fields are used, so it's better to not use IDs but unique class names. Rather than adding the ID I think we can trigger sortable directly on the ```container-repeatable-elements``` class.

Reply:
Ok for this !